app/services/twilio_api_service.rb

```diff
@@ -93,6 +97,7 @@ def send_twilio_message(sender_number, recipient_number, message, callback_url,
       body: message
     )
   rescue Twilio::REST::TwilioError => exception
+    metrics.increment("errors.#{communication_type}")
```

Comment:
Like @kpethtel mentioned, we can move these increments to `send_sms` and `send_whatsapp` so we don't have to change `TwilioApiService`'s signature. We could also pass in the `communication_type` to `send_twilio_message` as an argument.